dbsnp-importer/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=@eva.evapro.url@?currentSchema=@eva.dbsnp.schema@
```

Comment:
is the URL going to be like "server.url@?currentSchema=dbsnp_human"? Or is there a line jump missing?

Reply:
Yes, it is going to be like that (see [this SO post](https://stackoverflow.com/a/4820666)). The highlighting has messed up a bit with it 😄